charts/portal/values-production.yaml

```diff
@@ -20,7 +20,7 @@ global:
   legacyHostnames: false
   legacyDatabaseNames: false
   subdomainPrefix: dev-portal
-  helpPage: https://techdocs.broadcom.com/us/en/ca-enterprise-software/layer7-api-management/api-developer-portal/
+  helpPage: https://techdocs.broadcom.com/us/en/ca-enterprise-software/layer7-api-management/api-developer-portal/5-1/
```

Comment:
I tried with link '5-0-2' not working, so assuming 5-1 also will not work. So lets keep till 'https://techdocs.broadcom.com/us/en/ca-enterprise-software/layer7-api-management/api-developer-portal'

Reply:
5-1 is not yet published , I tested with 5-1test and it worked for me . If we keep till api-developer-portal - the links inside the portal - like landing page/home page and in api explorer page will not work.

Reply:
we change when 5-1 is available, for now lets remove.